zoom_utils/api_client.py

```diff
@@ -0,0 +1,85 @@
+import json
+
+import jwt
+import requests
+
+from zoom_utils import status
+from zoom_utils.constants import ZoomAPIEndpoint
+import logging as logger
+
+
+class ZoomAPIClient(object):
+    """
+    Helper class to consume Zoom API endpoints
+    """
+    VERSION = 'v2'
+    BASE_URL = 'https://api.zoom.us/' + VERSION
+    TOKEN_EXPIRY = '1496091963999'
```

Comment:
make it configurable, `os.getenv(VERSION,'v2')`

Reply:
Done!